pkg/Directory.Build.props

```diff
@@ -7,6 +7,15 @@
     <IncludeSymbols>false</IncludeSymbols>
   </PropertyGroup>
 
+  <!-- nuspec properties -->
+  <PropertyGroup>
+    <Authors>Microsoft</Authors>
+    <PackageLicenseUrl>https://github.com/dotnet/machinelearning/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>http://dot.net/ml</PackageProjectUrl>
+    <PackageIconUrl></PackageIconUrl>
+    <PackageReleaseNotes>https://github.com/dotnet/machinelearning/tree/master/Documentation/release-notes</PackageReleaseNotes>
```

Comment:
I have 2 suggested solutions to the package release notes problem listed here:

https://github.com/dotnet/corefx/issues/29314#issuecomment-387898104

I am taking the first approach for now (pointing to the root of the release-notes folder).  If we agree on the templating approach, we can change it.